stackslib/src/net/p2p.rs

```diff
@@ -5258,6 +5258,7 @@ impl PeerNetwork {
         );
         let mut network_result = NetworkResult::new(
             self.stacks_tip.block_id(),
+            self.stacks_tip.consensus_hash.clone(),
```

Comment:
`ConsensusHash` implements `Copy` so Clippy might complain about this once we enable it in CI. Can be:

```suggestion
            self.stacks_tip.consensus_hash,
```

Reply:
Fixed